src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/AuthService.kt

```diff
@@ -0,0 +1,50 @@
+package uk.gov.justice.digital.hmpps.manageusersapi.service
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.client.WebClient
+import org.springframework.web.reactive.function.client.WebClientResponseException
+import uk.gov.justice.digital.hmpps.manageusersapi.resource.Role
+import uk.gov.justice.digital.hmpps.manageusersapi.resource.RoleNameAmendment
+
+@Service
+class AuthService(
```

Comment:
Any reason to call this `AuthService` ? Should it be RoleService of something similar?

Reply:
Ah, it goes to Auth - I get it

Reply:
yep, the roleservice will be calling out to auth and prison, so pulled out